src/cdk/wizards/previewStateMachineCDKWizard.ts

```diff
@@ -0,0 +1,227 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as nls from 'vscode-nls'
+import * as picker from '../../shared/ui/picker'
+
+const localize = nls.loadMessageBundle()
+
+import {
+    MultiStepWizard,
+    WIZARD_GOBACK,
+    WIZARD_TERMINATE,
+    wizardContinue,
+    WizardStep,
+} from '../../shared/wizards/multiStepWizard'
+import { CdkAppLocation } from '../explorer/cdkProject'
+import { detectCdkProjects } from '../explorer/detectCdkProjects'
+import { AppNode } from '../explorer/nodes/appNode'
+import { ConstructNode } from '../explorer/nodes/constructNode'
+
+export interface CdkAppLocationPickItem {
+    label: string,
+    cdkApplocation: CdkAppLocation | undefined
+}
+
+export interface TopLevelNodePickItem {
+    label: string,
+    topLevelNode: ConstructNode | undefined
+}
+
+export interface ConstructNodePickItem {
+    label: string,
+    stateMachineNode: ConstructNode | undefined
+}
+
+interface PreviewStateMachineCDKWizardResponse {
+    cdkApplication: CdkAppLocationPickItem,
+    topLevelNode: TopLevelNodePickItem,
+    stateMachine: ConstructNodePickItem
+}
+
+export default class PreviewStateMachineCDKWizard extends MultiStepWizard<PreviewStateMachineCDKWizardResponse> {
```

Comment:
Any reason in particular to make this a `default` export?

Reply:
Nope! deleted the default export